Make example list of enabling policies (#1302)
publiccodenet/about: activities/understanding-our-users/enabling-policies.md

---
# SPDX-License-Identifier: CC0-1.0
# SPDX-FileCopyrightText: 2023 The Foundation for Public Code <info@publiccode.net>
type: Resource
---

# Enabling policies

> If there are policies supporting open source development it can be easier for civil servants to convince their peers that they should engage.

Reviewer: ```suggestion
Policies supporting open source development can make it easier for civil servants to convince their peers that they should engage.
```

> These policies can take many forms, from laws to internal policies.

Reviewer: This lets us avoid a 'policies can take many forms, including policies' recursive argument.
```suggestion
These policies can take many forms, from laws to internal guidance.
```

The clearer and more specific the policy is, the likelier it is that it will be a helpful argument.

> Some exampels (non-exhaustive list):

Reviewer 2: examples

---
# SPDX-License-Identifier: CC0-1.0
# SPDX-FileCopyrightText: 2023 The Foundation for Public Code <info@publiccode.net>
type: Resource
---

# Enabling policies

Policies supporting open source development can make it easier for civil servants to convince their peers that they should engage.
These policies can take many forms, from laws to internal guidance.

> The clearer and more specific the policy is, the more likely that it is that it will persuade skeptics.

Reviewer: ```suggestion
Clearer and more specific policies will be more likely to persuade skeptics.
```
